config/routes.rb

```diff
@@ -34,6 +34,10 @@
     invitations: "invitations",
   }
 
+  scope "/api", defaults: {format: :json} do
```

Comment:
I don't think you need this scope.

Reply:
Youre right, its not necessary.

Although think we should keep it this way. It gives clarity to the routes and url since no html pages will be served. It also allows us to define other endpoints with the same clarity and pattern in the future. Making it easier for developers to know where things go. 

Its a team decision though. I understand both sides.

Reply:
Bc the controller is not in the API scope as of now (I know it was at one point), I don't think this will work. I generally would prefer it be in an api module but since @hopsoft does not I think we need to remove this scope or it won't work if that makes sense.

Reply:
Yes, I did have it in a module previously. After a pair session with Nate he didnt think it was needed yet because our controllers are still manageable.  

The `scope` definition is different from `namespace` if that's what you're thinking of. Its similar to `namespace` but you define the attributes/settings. In our case nothing other than the actual route was defined. We have not defined `module: :api` as a parameter so it doesn't look for it.  

Again I'm fine with either way.

Reply:
oh I see. @hopsoft want to make a ruling?

Reply:
Let omit the scope for now as it's not used yet.

Reply:
Awesome :+1: Making the changes.